source/plugin/practices.rst

```diff
@@ -61,3 +70,59 @@ Plugin Interoperability
 =======================
 
 An explanation of how to communicate with other plugins, *TBA*.
+
+
+Bad Practices
+=============
+
+These should practices should be avoided. As they lead to memory leaks (``OutOfMemoryError``), lags or inconsistencies.
```

Comment:
Make this one sentence (...avoided, as ...).
They *can* lead to memory leaks - unless you're 100% certain they will?
Lag does not need to be a plural.